Core/ServeSync.Application/UseCases/EventManagement/EventCollaborationRequests/Dtos/EventCollaborationRequestCreateDto.cs

```diff
@@ -0,0 +1,46 @@
+using System.ComponentModel.DataAnnotations;
+using ServeSync.Application.UseCases.EventManagement.Events.Dtos.Events;
+using ServeSync.Domain.EventManagement.EventAggregate.Entities;
+using ServeSync.Domain.EventManagement.EventAggregate.Enums;
+using ServeSync.Domain.EventManagement.EventCollaborationRequestAggregate.ValueObjects;
+
+namespace ServeSync.Application.UseCases.EventManagement.EventCollaborationRequests.Dtos;
+
+public class EventCollaborationRequestDto
```

Comment:
Naming convention: Should add postfix Create to present the action for this dto -> EventCollaborationRequestCreateDto